src/main/java/com/faforever/client/game/CustomGamesController.java

```diff
@@ -252,4 +264,27 @@ protected void onHide() {
     // Hide all games to free up memory
     filteredItems.setPredicate(game -> false);
   }
+
+  public void onToggleSidePaneButtonClicked() {
+    boolean currentlyShowingSidePane = preferencesService.getPreferences().isShowGameDetailsSidePane();
+    preferencesService.getPreferences().setShowGameDetailsSidePane(!currentlyShowingSidePane);
+    preferencesService.storeInBackground();
+
+    setSidePane(!currentlyShowingSidePane);
+  }
+
+  private void setSidePane(boolean displayed) {
+    if (displayed) {
+      gameDetailPane.setVisible(true);
+      gameDetailPane.setManaged(true);
```

Comment:
Please bind manged to visible

Reply:
no, there's a different location in the code that just sets the pane to invisible (if no game is selected) while keeping it managed.

Reply:
Still solve it with bindings. You can incorporate that into the binding most likely

Reply:
you could if selected game would be a property. There's a reason the old code does it that way.

Reply:
ok, that worked, had to expose the game property as read only property